### ISSUE TITLE
php 5.3 remains instable

### ISSUE THREAD

**Issue by Puddingboy** [AnimalDesign/bolt-translate#12]
Allow me to first explain something from my side: the IT guy I was referring to is the one running php 5.3 and I'm the one running 5.4. He was gone past fryday so I couldn't contact him to ask if things were solved. While I'm developing on the main page in the test envirement, our IT guy is just checking if bolt with the plugins is running stable on the main system. I wouldn't have been able to contact him in his absence even if I wanted to.

These are the recent issues he's encountering:
```
Error: syntax error, unexpected '['
File: extensions/vendor/animal/translate/src/Controller/AsyncController.php
Line: 16
```

```
 $router->match('/get', [$this, 'getTranslationAction'])
            ->method('GET');
```

```
Error: syntax error, unexpected '['
File: extensions/vendor/animal/translate/src/Content/LocalizedContent.php
Line: 106

```
`$defaultLocaleSlug = reset($locales)['slug'];`

```
Error: syntax error, unexpected '['
File: extensions/vendor/animal/translate/src/Content/LocalizedContent.php
Line: 126
```

`$this->delocalizedValues = [];`

```
Error: Using $this when not in object context
File: extensions/vendor/animal/translate/src/Content/LocalizedContent.php
Line: 110
```

`return $e['slug'] == $this->app['request']->get('_locale');`

There may be more issues in there, but this is as far as we got.

I hope you're not taking this as critique. It was actually your plugin that made the biggest difference in deciding if using bolt cms was an option in this project. Your extension arrived just at the right moment and really your timing couldn't have been better. 

I wish I could be of bigger help in terms of helping you debug and providing support, but I've always been terrible at understanding code written by others. I'm also still figuring out bolt itself in many ways. Just know that you are a doing an awesome job.

**Comment by SahAssar** [comment]
I finally "capitulated" :wink: and got found some shared hosting that has php 5.3 to test on (I use CentOS 7 on my servers and php 5.3 is uninstallable on that) and fixed all the bugs I could find. You also had some issues with the default content displaying in other languages, right? Cause that should be fixed too, that was a issue with an `array_filter` that didn't appear for me in php 7. Could you install dev-master on your site and see if it fixes this issue for you? I don't really want to fire out releases unless issues are actually fixed in them. You install developer versions of extensions by setting [this](https://github.com/bolt/bolt/blob/release/2.2/app/config/config.yml.dist#L391) to "dev" and then selecting "dev-master" when installing.

I'm taking this as critique, but the good, constructive kind, and I appreciate that you have the patience to help me work out all these bugs. :smile:

I'm very glad that I was a factor in you deciding that bolt was right for you!

**Comment by Puddingboy** [comment]
I fear the last submit as a dev-master did not register yet or not properly so he had to work his way around it. To the point: this seems to have fixed the issue. What would tabs etc would you like us to check before you'll submit this as an update?

**Comment by SahAssar** [comment]
Seems good enough for me, It worked on my php 5.3 install and it works on yours, which is one more test environment than we had before :) I'm tagging a release now, so it will be in bolt extensions in an hour or so.

I'm also closing this since it should be fixed by b32f16c648e1fa22df33b5979c5c83f8cf8d3443, if you have more issues please let me know :)
